core/templates/dev/head/directives.js

```diff
@@ -164,3 +164,19 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
     }
   };
 }]);
+
+oppia.directive('triggerClickOnEnter', function () {
+  return {
+    restrict: 'A',
```

Comment:
@srikar0896 , Custom directives should be 'Element' type only. Can you change it to `restrict: 'E'` and make necessary changes. It will also pass the failing tests.

Reply:
@srikar0896 I have a question about this directive. It looks like you want to "click" a link when the keyboard focus is moved to that link and the Enter key is pressed.

My question is, why doesn't this happen automatically? E.g. on this GitHub page when I tab around to a link and then hit Enter, I get redirected to that page. This seems like the standard behaviour on HTML pages. Why do we need a special directive for this in Oppia?

Reply:
But in this case we are not opening a page, on click we are opening this modal.So, I have written a directive which will trigger click on Enter keypress.
![image](https://user-images.githubusercontent.com/17567875/38170741-7fd26d3a-35aa-11e8-9aaa-f6354bb67043.png)

Reply:
@vibhor98 Thanks, will do it.

Reply:
No, It works if the anchor tag has the href attribute.But our case there isn't.The workaround would be to add an empty href.But I thought it would be a precarious hack, So added a directive to handle and this.
Please correct me if I am wrong

Reply:
Thanks for the explanation! I think an empty `href` is fine.

Reply:
Done.Removed the directive and added empty href.